CHANGES.rst

```diff
@@ -11,7 +11,7 @@ Bugfix:
  -
 
 API Change:
- -
+ - Parameter historyVisibility removed from MxSession.createRoom(). It had no effect.
```

Comment:
Please add a line:
` - new API: CreateRoomParams.setHistoryVisibility(String historyVisibility) to force the history visibility during Room creation`